### PR TITLE
add a new function called delete_book

### PR DESCRIPTION
**Pull request by davidyunda**
add a new option that allows the user to delete a book by asking for the book ID and display an error message if the book is not found
 
@wyatt-geiger

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -65,9 +66,16 @@ def change_read():
     book.read = new_read 
     book.save()
     
+def delete_book():
+    try:
+        search_id = ui.ask_question('Enter Book ID:')
+        match_id = store.get_book_by_id(search_id)
+        store._delete_book(match_id)
```
Comment (claraj) [inline main.py:73]: Don't call any _ methods directly. There's another method that's already in bookstore to use.

Comment (claraj) [inline main.py]: Don't call the store method directly - see note here. Can you ask the book object to delete itself? https://github.com/DashKazak/readinglist/blob/e7b462ef3986b08f26a62c290fb0ae277cc53ae3/bookstore.py#L82-L91

File: main.py [hunk 2]
```diff
@@ -65,9 +66,16 @@ def change_read():
...
+    except:
+        print('Error: Book Not Found')
 
 def quit_program():
-    ui.message('Thanks and bye!')
+    ui.message('Thanks for using the book reading list!')
```
Comment (claraj) [inline main.py]: this is from another feature and should not be in this branch

Comment (claraj) [reply main.py]: Make sure you check out main, before creating a new branch for another feature

**Comment by DashKazak** [comment]
It looks good to me. (we discussed this one during team meeting)

**Pull request merged**